### PR TITLE
modifications on PaddleVideo to enable universality of codes

### PR DESCRIPTION
**Pull request by SunGaofeng**
1,  modify codes to enable universality for different models
2, keep consistent with other codebases under PaddlePaddle/models

### CODE REVIEW

File: PaddleCV/PaddleVideo/data/dataset/kinetics_labels.json [hunk 1]
```diff
@@ -0,0 +1 @@
...iving", "squat", "sticking_tongue_out", "stomping_grapes", "stretching_arm", "stretching_leg", "strumming_guitar", "surfing_crowd", "surfing_water", "sweeping_floor", "swimming_backstroke", "swimming_breast_stroke", "swimming_butterfly_stroke", "swing_dancing", "swinging_legs", "swinging_on_something", "sword_fighting", "tai_chi", "taking_a_shower", "tango_dancing", "tap_dancing", "tapping_guitar", "tapping_pen", "tasting_beer", "tasting_food", "testifying", "texting", "throwing_axe", "throwing_ball", "throwing_discus", "tickling", "tobogganing", "tossing_coin", "tossing_salad", "training_dog", "trapezing", "trimming_or_shaving_beard", "trimming_trees", "triple_jump", "tying_bow_tie", "tying_knot_(not_on_a_tie)", "tying_tie", "unboxing", "unloading_truck", "using_computer", "using_remote_controller_(not_gaming)", "using_segway", "vault", "waiting_in_line", "walking_the_dog", "washing_dishes", "washing_feet", "washing_hair", "washing_hands", "water_skiing", "water_sliding", "watering_plants", "waxing_back", "waxing_chest", "waxing_eyebrows", "waxing_legs", "weaving_basket", "welding", "whistling", "windsurfing", "wrapping_present", "wrestling", "writing", "yawning", "yoga", "zumba"]
```
Comment (shippingwang) [inline PaddleCV/PaddleVideo/data/dataset/kinetics_labels.json:1]: What's this file used for?

File: PaddleCV/PaddleVideo/inference_model.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    parser.add_argument(
+        '--use_gpu',
+        type=ast.literal_eval,
+        default=True,
+        help='default use gpu.')
+    parser.add_argument(
+        '--weights',
+        type=str,
+        default=None,
+        help='weight path, None to use weights from Paddle.')
```
Comment (shippingwang) [inline PaddleCV/PaddleVideo/inference_model.py]: "None to use weights from paddle" What's this meaning?

File: PaddleCV/PaddleVideo/data/dataset/kinetics_labels.json (hunk 1 above)
Comment (SunGaofeng) [reply PaddleCV/PaddleVideo/data/dataset/kinetics_labels.json:1]: this is for inference of tsn tsm stnet nonlocal, mapping label id to category name.

File: PaddleCV/PaddleVideo/inference_model.py (hunk 2 above)
Comment (SunGaofeng) [reply PaddleCV/PaddleVideo/inference_model.py]: It means that program will download weights automatically when None is given.
Yes, I should change the description.

**Review by shippingwang — APPROVED** [review]
good job, approved

**Pull request merged**